### PR TITLE
PR for 570 user profile image release prep

### PR DESCRIPTION
**Pull request by qbi11y**
Contributes to #570 

Updated code based on release review guidelines

#### What did you change?
`UserProfileImage.js`
`UserProfileImage.test.js`
`_userProfileImage.scss`
`UserProfileImage.stories.js`
`../index.scss

#### How did you test and verify your work?
All test passed

### CODE REVIEW

File: packages/cloud-cognitive/src/components/UserProfileImage/_userProfileImage.scss [hunk 1]
```diff
@@ -5,16 +5,17 @@
...
-$x-large: $spacing-10; //64px
-$large: $spacing-07; //32px
-$medium: $spacing-06; //24px
-$small: 1.25rem; //20px
-$x-small: $spacing-05; //16x
+$xl: $spacing-10; //64px
+$lg: $spacing-07; //32px
+$md: $spacing-06; //24px
+$sm: 1.25rem; //20px
+$xs: $spacing-05; //16x
```
Comment (lee-chase) [inline packages/cloud-cognitive/src/components/UserProfileImage/_userProfileImage.scss:18]:
```suggestion
$sizes: (
  xl: $spacing-10, //64px
  lg: $spacing-07, //32px
  md: $spacing-06, //24px
  sm: 1.25rem, //20px
  xs: $spacing-05 //16x
);

@mixin size($size: 'md') {
    $_size: map-get($sizes, $size);
    width: $_size;
    height: $_size;
}
```

**Review by lee-chase — CHANGES_REQUESTED** [review]
One suggestion just to make maintenance a little easier.

**Review by matthewgallo — APPROVED** [review]
Looks like the requested changes from @lee-chase have been addressed and added in. Looks good @qbi11y!

File: packages/cloud-cognitive/src/components/UserProfileImage/_userProfileImage.scss (hunk 1 above)
Comment (matthewgallo) [reply packages/cloud-cognitive/src/components/UserProfileImage/_userProfileImage.scss]: Added in latest [commit](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/572/commits/942ee99c00b434a9d3d32b057fcb099b7d78df4d).

**Pull request merged**